basiclti/web-ifp/src/bundle/vm/pick.vm

```diff
@@ -0,0 +1,57 @@
+<script type="text/javascript">
+	focus_path = [ "title" ];
+</script>
+
+<div class="portletBody">
+<script type="text/javascript" src="/library/js/headscripts.js"></script>
+<script>
+	includeLatestJQuery('web-ifp pick.vm');
+</script>
+	<h3>
+		$tlang.getString("gen.custom")
+	</h3>
+	#if ($alertMessage)
+	<div class="sak-banner-error">
+	$tlang.getString("gen.alert") $alertMessage
+	</div>
+
+	#end
+	<form action="$actionUrl" method="post" name="customizeForm" >
+
+#if ($tools && $tools.size() > 0)
+        <p style="clear:all;">
+            <label for="tool_id" style="display:block;float:none;">
+                <span style="color:#903;font-weight:bold" title="Tool Title">*</span>
+        $tlang.getString("content.select.tool")
+            </label>
+   <select id="tool_id" name="tool_id"">
+      <option value="--none--">$tlang.getString("content.select.one")</option>
+    #foreach ($tool in $tools)
+        #set( $selected = "" )
+        #if ( $tool_id && $tool_id.toString().equals($tool.get("id").toString()))
+            #set( $selected = "selected" )
+        #end
+               <option value="$tool.get("id")" $selected > $tool.get("title")</option>
+    #end
+   </select>
+</p>
+<p class="act">
+<input type="submit" accesskey ="s" class="active" name="$doChoose" value="$tlang.getString('gen.save')" />
+<input type="submit" accesskey ="x" name="$doCancel" value="$tlang.getString('gen.cancel')" />
+</p>
+#else
+<p>$tlang.getString("content.no.tools")</p>
+<p>
+<p class="act">
+<input type="submit" accesskey ="x" name="$doCancel" value="$tlang.getString('gen.cancel')" />
+</p>
+#end
+	</form>
+
+<script type="text/javascript">
+$(document).ready(function () {
```

Comment:
we should stop using jquery's ready
`document.addEventListener("DOMContentLoaded", function() { });`
see https://stackoverflow.com/questions/2304941/what-is-the-non-jquery-equivalent-of-document-ready

Reply:
OK.